tests/scripts/all.sh

```diff
@@ -438,8 +438,40 @@ if uname -a | grep -F x86_64 >/dev/null; then
 msg "build: i386, make, gcc" # ~ 30s
 cleanup
 CC=gcc CFLAGS='-Werror -Wall -Wextra -m32' make
+
+msg "build: gcc, force 32-bit compilation"
+cleanup
+cp "$CONFIG_H" "$CONFIG_BAK"
+scripts/config.pl unset MBEDTLS_HAVE_ASM
+scripts/config.pl unset MBEDTLS_AESNI_C
+scripts/config.pl unset MBEDTLS_PADLOCK_C
+CC=gcc CFLAGS='-Werror -Wall -Wextra -DMBEDTLS_HAVE_INT32' make
```

Comment:
The options MBEDTLS_HAVE_INT32, MBEDTLS_HAVE_INT64 and MBEDTLS_TYPE_UDBL are not in the config.h so the script does not work in that case. Actually, I am not sure whether we need to add these options to the config.h, MBEDTLS_HAVE_INT32 was not there previously, so thats why I did not add them